src/pages/config/ConfigDetailsInsights.tsx

```diff
@@ -0,0 +1,75 @@
+import { useParams, useSearchParams } from "react-router-dom";
+import { InfoMessage } from "../../components/InfoMessage";
+import { SearchLayout } from "../../components/Layout";
+import ConfigSidebar from "../../components/ConfigSidebar";
+import { ConfigsDetailsBreadcrumbNav } from "../../components/BreadcrumbNav/ConfigsDetailsBreadCrumb";
+import {
+  useGetConfigByIdQuery,
+  useGetConfigInsights
+} from "../../api/query-hooks";
+import { Head } from "../../components/Head/Head";
+import TabbedLinks from "../../components/Tabs/TabbedLinks";
+import { useConfigDetailsTabs } from "../../components/ConfigsPage/ConfigTabsLinks";
+import { ConfigTypeInsights } from "../../components/ConfigInsights";
+import InsightsDataTable from "../../components/InsightsDataTable";
+
+export function ConfigDetailsInsights() {
+  const { id } = useParams();
+  const [params, setParams] = useSearchParams();
+
+  const {
+    data: configInsights = [],
+    isLoading,
+    refetch
+  } = useGetConfigInsights<ConfigTypeInsights[]>(id ?? "");
+
+  const { data: configItem, error: itemError } = useGetConfigByIdQuery(id!);
+
+  const configTabList = useConfigDetailsTabs();
+
+  if (itemError) {
+    const errorMessage =
+      typeof itemError === "symbol"
+        ? itemError
+        : itemError?.message ?? "Something went wrong";
+
+    return <InfoMessage message={errorMessage} />;
+  }
+
+  return (
+    <>
+      <Head prefix={configItem ? `Config Insights - ${configItem.name}` : ""} />
```

Comment:
May be just default to Config Insights when config insight is not available

Reply:
Done @mainawycliffe